.pre-commit-config.yaml

```diff
@@ -19,7 +19,7 @@ repos:
     rev: v2.14.1
     hooks:
     - id: pylint
-  - repo: https://gitlab.com/pycqa/flake8
+  - repo: https://github.com/pycqa/flake8
```

Comment:
nice catch - I wonder how the gitlab url got in there (it doesn't even resolve for me?)